devTools/cardCreator/cards_to_import/BeastboundSavage.minion.card.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "Beastbound Savage",
+  "description": "Celerity. After attacking, deal 2 damage to all surrounding enemies",
```

Comment:
add a new line character after Celerity.
"Celerity. \n After attacking, deal 2 damage to all surrounding enemies"